### DISCUSSION TITLE
Permissions to create repositories for specific teams

### DISCUSSION THREAD

**Discussion by fleed** [community/community#46502] [New to GitHub] [answered]
Hello!
In the settings for an organization I can define the policy for creation or repositories for all members.
I disabled it, but I would like to add "exceptions" so that members of specific teams can create new repositories.
I cannot find a way to define these "exceptions". Is it not possible at all, or am I just not looking at the correct place?

Thank you in advance

**Comment by shayan1234554321** [comment, marked as answer]
Unfortunately ☢️ , it's not currently possible to set repository creation permissions for specific teams within a GitHub organization. The repository creation policy for an organization is applied to all members ❤️‍🩹 , regardless of their team membership.

If you need specific members to be able to create repositories, you could consider granting them administrative access to the organization or adding them as collaborators ✌️  to the repositories they need to work on.

However, if you are looking for a way to control repository creation for your organization, I suggest reaching out to GitHub Support 🤝 and asking if they have any plans to implement this feature in the future. They may be able to provide you with more information or suggest alternative solutions.

**Reply by fleed** [reply]
I will ask if the feature is planned. Thank you

**Comment by Feuikkes** [comment]
Floating and crucifying this is my growth filing is next
